tactics/hints.ml

```diff
@@ -1028,7 +1028,7 @@ let remove_hint dbname grs =
 type hint_action =
   | CreateDB of bool * TransparentState.t
   | AddTransparency of evaluable_global_reference hints_transparency_target * bool
-  | AddHints of hint_entry list
+  | AddHints of bool (* superglobal? *) * hint_entry list
```

Comment:
I think we are on 4.03+ now, we should start naming fields instead of using comments
`AddHints of { superglobal : bool; hints : hint_entry list }` should do the trick

Reply:
Done.